src/MakeBusy/Kazoo/SDK.php

```diff
@@ -9,6 +9,21 @@
 
 use \MakeBusy\Common\Configuration;
 
+function substitute_vars($object) {
```

Comment:
also it should reside inside the class, not in the global scope. maybe a static function on the SDK class?